config/kernel.m4

```diff
@@ -2,6 +2,7 @@ dnl #
 dnl # Default ZFS kernel configuration
 dnl #
 AC_DEFUN([ZFS_AC_CONFIG_KERNEL], [
+	AM_COND_IF([BUILD_LINUX], [
```

Comment:
Missing indentation.  Just a question, but do you expect you'll ever need/want equivalent functionality for FreeBSD?

Reply:
Indentation fixed.

Reply:
> do you expect you'll ever need/want equivalent functionality for FreeBSD?

I don't think so, our kernel build system manages configuration adequately enough for our needs.